Example/InstanceID/Tests/FIRInstanceIDTest.m

```diff
@@ -165,7 +165,7 @@ - (void)testSharedInstance {
   // The shared instance relies on the default app being configured. Configure it.
   FIROptions *options = [[FIROptions alloc] initWithGoogleAppID:kGoogleAppID
                                                     GCMSenderID:kGCMSenderID];
-  options.APIKey = @"api-key";
+  options.APIKey = @"APIKeyWithValidFormat1920404502934lksfj";
```

Comment:
All API keys start with the String "`AIzaSy`". \
Thus, even though we currently do *not* check for this, I recommend to follow this format.
Also, I strongly recommend to use all special characters that are allowed, particularly "`-`" and "`_`" to make sure future changes cover these characters.

How about this API key:
"`AIzaSy-ApiKeyWithValidFormat_0123456789`"